dali/pipeline/operators/reader/parser/parser_test.cc

```diff
@@ -66,22 +67,23 @@ TYPED_TEST_SUITE(ParserTest, TestTypes);
 
 TYPED_TEST(ParserTest, BasicTest) {
   const int H = 64, W = 64, C = 3;
-  int *data = new int[3 + H*W*C];
+  std::vector<int> data;
```

Comment:
```suggestion
  std::vector<int> data(3 + H*W*C);
```
and remove next line

Reply:
Done